config.example.yml

```diff
@@ -1,6 +1,12 @@
+#create a bot to obtain tokens: https://discordapp.com/developers/applications/ 
 clientid:
 clientsecret:
 bottoken:
-http:
+errorlog:
+adminuser:
+shards: 1 #needs at least one shard
```

Comment:
This is a bug - I didn't want you to have to specify the count if you only needed one.